CMakeLists.txt

```diff
@@ -755,12 +784,9 @@ if(LIBRE_BUILD_SHARED)
   )
 endif()
 
-install(FILES cmake/re-config.cmake
```

Comment:
Removing `re-config.cmake` breaks baresip and other apps.
Its currently used to keep compile definitions aligned.

Reply:
I am not familiar with the baresip project. So this comment is from an outside perspective.

The installed `cmake/re-config.cmake` caused confusion in reviewing the vcpkg port. AFAIU it has no value to the *installed* (lib)re project. It doesn't capture the settings that were passed to the build, and it doesn't capture the values which were detected during the build. It doesn't fully behave like regular config packages: It can make the build fail (from `find_package(... REQUIRED)`) instead of letting find_package continue to search for it in other locations. It is really more a script file which could be used via `include(...)` (and `CMAKE_MODULE_PATH`) than a package to be used via `find_package(...)`. However, I acknowledge the comfort of `find_package` 's config search for the desired shared use.

From that POV, it shouldn't be installed in this form with (lib)re. This sounds harsh, so let me add loose suggestions:
- Use a different name to mark it as an independent component, e.g. baresip-common or BaresipSetup.
- Burn captured configuration results into the installed config.
  (It might still use `find_dependency(OpenSSL)` etc.)